packages/Filter/src/helpers/testers.js

```diff
@@ -17,6 +17,16 @@ const processDate = (momentParsingFormat, value, testValue, testFunction) => {
   return false;
 };
 
+const isOneOf = (testValue, acceptableValues) => {
+  if (testValue === "") return true;
```

Comment:
`testValue` is the value in `data` source, `acceptableValues ` is the array value for multiselect, by default it's `[]` so I thought it should be `if (acceptableValues.length === 0) return true;`

Reply:
You are correct